content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -141,12 +141,30 @@ const [findRawData, aggregateRawData, commandRawData] =
 
 </TabbedContent>
 
-Instead of immediately awaiting the result of each operation when it's performed, the operation itself is stored in a variable first which later is submitted to the database via a method called `$transaction`. Prisma Client will ensure that either all three `create`-operations succeed or none of them succeed.
+Instead of immediately awaiting the result of each operation when it's performed, the operation itself is stored in a variable first which later is submitted to the database with a method called `$transaction`. Prisma Client will ensure that either all three `create`-operations succeed or none of them succeed.
 
 > **Note**: Operations are executed according to the order they are placed in the transaction. Using a query in a transaction does not influence the order of operations in the query itself.
 >
 > Refer to the 📖 [transactions guide](/guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api) for more examples.
 
+The sequential transaction API has a second parameter with the following optional configuration option:
```

Comment:
Should we mention that this is available only starting from 4.4.0? I saw we do this for other APIs in some places?

Reply:
Good point - I did mention it in the dedicated **Isolation levels** section, but it would be better to also mention that here. Thanks!